pkg/nss/manager_windows.go

```diff
@@ -4,17 +4,62 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
+// getFirefoxProfilePath returns the path to the Firefox default profile directory.
+// It will search for the first directory that ends with ".default-release" which is the directory that contains the NSS database.
+func getFirefoxProfilePath() (string, error) {
+	appData := os.Getenv("APPDATA")
+	if appData == "" {
+		return "", fmt.Errorf("failed to get APPDATA environment variable")
+	}
+
+	firefoxProfilesDir := filepath.Join(appData, "Mozilla", "Firefox", "Profiles")
+	_, err := os.Stat(firefoxProfilesDir)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return "", fmt.Errorf("failed to find Firefox profiles directory: %w", err)
+		}
+		return "", fmt.Errorf("failed to check Firefox profiles directory: %w", err)
+	}
+
+	profiles, err := os.ReadDir(firefoxProfilesDir)
+	if err != nil {
+		return "", fmt.Errorf("failed to read Firefox profiles directory: %w", err)
+	}
+
+	var defaultProfileDir string
+	for _, profile := range profiles {
+		if profile.IsDir() && strings.HasSuffix(profile.Name(), ".default-release") {
+			defaultProfileDir = profile.Name()
+			break
+		}
+	}
+
+	if defaultProfileDir == "" {
+		return "", fmt.Errorf("failed to find default Firefox profile directory")
+	}
+
+	nssDBPath := filepath.Join(firefoxProfilesDir, defaultProfileDir)
+
+	return nssDBPath, nil
+}
+
 // defaultNSSDatabasePaths returns the known NSS databases directories of a Windows operating system.
 func defaultNSSDatabasePaths() ([]string, error) {
 	appData := os.Getenv("APPDATA")
 	if appData == "" {
 		return nil, fmt.Errorf("failed to get APPDATA environment variable")
 	}
 
-	nssDBPaths := []string{
-		filepath.Join(appData, "Mozilla\\Firefox\\Profiles"),
+	nssDBPaths := []string{}
+
+	firefoxDBPath, err := getFirefoxProfilePath()
+	if err == nil {
+		// If we found the Firefox profile directory, we add it to the list of NSS databases.
+		// If we didn't find it, we just ignore it and continue because it means that Firefox is not installed.
+		nssDBPaths = append(nssDBPaths, firefoxDBPath)
 	}
```

Comment:
Why aren't you simply doing:
```suggestion
	nssDBPaths := []string{
		filepath.Join(appData, "Mozilla", "Firefox", "Profiles", "*"),
	}
```

Reply:
Because certutil returns an error when using `*`. I don't know if it's because of the other profile in the directory, or just because it does not find the "*" directory and so, fails.

![image](https://github.com/massalabs/station/assets/44696378/c4846f78-11c7-4793-a9bf-0ef709bf1cd0)

Reply:
It looks like it's because certutil searches for the `*` directory. 
I tried with a directory name that does not exists and got the same error

Reply:
Why is it working for linux ? expander [here](https://github.com/massalabs/station/blob/f2e244862f63372c0f942440b7ce8371ed3ce768/pkg/nss/manager.go#L116) should accept a *